### ISSUE TITLE
Error  Cannot read property 'replace' of undefined after upgrade

### ISSUE THREAD

**Issue by khayalan-mathew** [nestjs/nest-cli#466]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
Right now the build errors with following error code:
` Error  Cannot read property 'replace' of undefined`

Using `--webpack` fixes the build, so its apparently an issue with the tsc option.
Happens when updating from 6.11.2 to 6.11.3

## Expected behavior
Build to succeed

## Minimal reproduction of the problem with instructions
https://github.com/khayalan-mathew/nestjs-repro-build-error
Use yarn to install dependencies and then yarn build

## What is the motivation / use case for changing the behavior?
Making builds not break.

## Environment

```
 _   _             _      ___  _____  _____  _     _____
| \ | |           | |    |_  |/  ___|/  __ \| |   |_   _|
|  \| |  ___  ___ | |_     | |\ `--. | /  \/| |     | |
| . ` | / _ \/ __|| __|    | | `--. \| |    | |     | |
| |\  ||  __/\__ \| |_ /\__/ //\__/ /| \__/\| |_____| |_
\_| \_/ \___||___/ \__|\____/ \____/  \____/\_____/\___/


[System Information]
OS Version     : macOS Mojave
NodeJS Version : v10.16.3
YARN Version    : 1.19.1
[Nest Information]
platform-express version : 6.7.2
swagger version          : 3.1.0
common version           : 6.7.2
core version             : 6.7.2
```

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by khayalan-mathew** [comment]
Gonna try to provide one within this week.

**Comment by minzojian** [comment]
same error

**Comment by khayalan-mathew** [comment]
Made reproduction repo:
https://github.com/khayalan-mathew/nestjs-repro-build-error

I use yarn as a package manager

**Comment by khayalan-mathew** [comment]
> i found same error
> but after i compared the default project files i found there is one missed line in tsconfig.json in my own project
> 
> `"baseUrl": "./",`
> 
> add it into you tsconfig.json see as if it works

Works for me :)

**Comment by db306** [comment]
Hello, I had the same issue. Adding that line to tsconfig solved the issue.

Best regards,

**Comment by kamilmysliwiec** [comment]
Fixed in 6.12.4 release :)
